src/TritonRoute/src/dr/FlexDR_init.cpp

```diff
@@ -1032,10 +1041,7 @@ void FlexDRWorker::initNet_termGenAp_new(const frDesign* design, drPin* dPin)
           } else {
             yLoc = (yl(pinRect) + yh(pinRect)) / 2;
           }
-          bool halfOnTrackFound = !yLocs.empty() || !xLocs.empty();
-          if (currLayerNum >= BOTTOM_ROUTING_LAYER && (!noJog || 
-                halfOnTrackFound) && (!isInitDR() || (xLoc != xh(routeRect) || 
-                yLoc != yh(routeRect)))) {
+          if (!isInitDR() || (xLoc != xh(routeRect) || yLoc != yh(routeRect))) {
```

Comment:
Please explain the change in condition here - dropping the layer/job/halfTrack conditions.

Reply:
This is a revert of my temp fix to make gf12/coyote pass (and that probably caused the "still dont have aps" error reported by Louii). So the current code is the default one.